tests/test_packages/test_protocols/test_fipa.py

```diff
@@ -303,3 +293,132 @@ def test_dialogues():
     dialogues.dialogue_stats.add_dialogue_endstate(FIPADialogue.EndState.DECLINED_CFP, is_self_initiated=False)
     assert dialogues.dialogue_stats.self_initiated == {FIPADialogue.EndState.SUCCESSFUL: 1, FIPADialogue.EndState.DECLINED_PROPOSE: 0, FIPADialogue.EndState.DECLINED_ACCEPT: 0, FIPADialogue.EndState.DECLINED_CFP: 0}
     assert dialogues.dialogue_stats.other_initiated == {FIPADialogue.EndState.SUCCESSFUL: 0, FIPADialogue.EndState.DECLINED_PROPOSE: 0, FIPADialogue.EndState.DECLINED_ACCEPT: 0, FIPADialogue.EndState.DECLINED_CFP: 1}
+    assert dialogues.dialogues_as_seller is not None
+    msg = FIPAMessage(message_id=0,
+                      dialogue_reference=(str(0), ''),
+                      target=1,
+                      performative=FIPAMessage.Performative.ACCEPT)
+    msg.counterparty = "opponent"
+    result = dialogues.is_permitted_for_new_dialogue(msg)
+    assert result is False
+
+
+def test_dialogues_is_belonging_to_registered_dialogue():
+    """Test if the dialogue is a part of an other dialogue."""
+    dialogues = FIPADialogues()
+    result = dialogues.create_opponent_initiated(dialogue_opponent_addr="opponent", dialogue_reference=(str(0), ''), is_seller=True)
```

Comment:
This would be called if you receive a message from an opponent. so first I would create a message object, then from it call this, then send a response message..